AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/flowMisc.js

```diff
@@ -678,4 +678,54 @@ export function addSubnetSelectionDropDownView(siteInfo, diagProvider, flowMgr,
             dropdownView.dropdowns.length = 0;
             dropdownView.dropdowns.push(subscriptionDropdown);
         });
+}
+
+export async function checkDaaSExtApiAsync(diagProvider) {
+    var params = [];
+    params.push("api-version=2015-08-01");
+    var daasVersionInfo=await diagProvider.getDaaSExtApiAsync("daasversion",params);
+    var isDaasExtAccessible;
+    if(daasVersionInfo.status == "200") {
+        isDaasExtAccessible = true;
+    }
+    else {
+        isDaasExtAccessible = false;
+    }
+
+    var isDaasNew = false;
+    if(isDaasExtAccessible == true) {
+        var currentDaasVersion = daasVersionInfo.body.Version;
+        if (checkDassNewVersion(currentDaasVersion) >= 0) {
+            isDaasNew = true;
+        }
+        else {
+            isDaasNew = false;
+        }
+    }
+
+    return { "IsDaasExtAccessible": isDaasExtAccessible, "IsDaasNew": isDaasNew }
+}
+
+function checkDassNewVersion(currentDaasVersion){ 
```

Comment:
nit: typical naming convention for a comparison type functions is "isNewerDaasVersion" or "isDaaSVersionNewer" and should return a boolean instead of comparison result as we don't care if its older than reference version but rather that its newer or not.

Reply:
fixed